manifesto.html

```diff
@@ -13,7 +13,7 @@ <h1 class="cover-heading">Bądźmy życzliwi ❤</h1>
     </p>
 
     <p class="lead">
-        Ta różnorodność sprawia, że dzięki różnym perspektywom i współpracy
+        Te różnice sprawiają, że dzięki różnym perspektywom i współpracy
```

Comment:
Paragraf wyżej jest już użyte słowo "różnice". O ile w języku polskim słowa "różnica" i "różnorodność" rzeczywiście mają przynajmniej zbliżone do siebie znaczenie, to definicje ich angielskojęzycznych odpowiedników (odpowiednio "difference" i "diversity") wskazują na różne zakresy, które te słowa obejmują - "difference" jest używane bardziej w kontekście pojedynczych rzeczy różniących się od reszty, zaś "diversity" obejmuje jakiś większy zbiór różnych rzeczy/cech/opinii.